docs/source/user/commands.rst

```diff
@@ -18,3 +29,12 @@ The command palette can be accessed from the View menu or using the keyboard sho
 ``Command/Ctrl Shift C``.
 
 The command palette can be displayed in the sidebar by adding ``'modal': false`` to the Settings.
+
+Commands List
+^^^^^^^^^^^^^
+
+The table below display the list of commands and the associated keyboard shortcuts. The
+unique identifier is useful if you want to customize the menus, the toolbars or the keyboard shortcuts.
```

Comment:
Also I get often confused with how the shortcuts are documented for various operating systems. The generated list shows  always `Ctrl`. As a Mac user, do I need to translated that to `Cmd`? Some context or legend would be useful here.

Reply:
I don't know how to solve that one. The reason it is always showing `Ctrl` is because we are using Linux as OS to get the command list. Therefore the shortcut is computed for that OS. This means `Cmd` can be used instead of `Ctrl` (they may be some exceptions). But I definitely can add a note about that.